lib/puppet/reports/datadog_reports.rb

```diff
@@ -69,16 +79,20 @@ def process
         event_title = "Puppet failed on #{@msg_host}"
         alert_type = "error"
         event_priority = "normal"
+        check_status = 2
```

Comment:
Did you mean to put 2 instead of 1 here?

Reply:
Yes. 2 means critical state for this check.